.changeset/young-badgers-yawn.md

```diff
@@ -0,0 +1,11 @@
+---
+'@xstate/analytics': patch
```

Comment:
This changeset is OK - but atm we dont intend to release all packages, could you include only test package here (we publish core xstate, fsm and test packages right now)? Changes in all package.jsons should be kept.

Reply:
I believe I should be guarded by the `private` field in those packages?

Reply:
Anyway, I've just removed all packages in here that were marked as private in their package.json

Reply:
Private guards us only from publishing to npm but i want to avoid other release stuff as well - version bumping, changelog generation (covered by changesets). I plan to clean this up soon so we dont have to worry about it - but atm it is what it is.

Reply:
Do you just want me to remove the change set then? 😛

Reply:
Nope, right now I'd like to just keep `@xstate/test` in it - nothing more. I understand this is confusing, that's why I intend to clean things up in the near future so we can just assume that everything is publishable.

Reply:
👍